tutorial-content/content/weighting-schemes.md

```diff
@@ -255,6 +273,19 @@ Start by throwing away extraneous data, by cropping the LandScan to, say,
 landscan <- crop(landscan, extent(-126, -66, 24, 50))
 ```
 ````
+````{tab-item} Python
+```Python
+from rasterio.windows import from_bounds
+
+xmin, xmax, ymin, ymax = -126, -66, 24, 50
+
+# Calculate the window to crop
+window = from_bounds(xmin, ymin, xmax, ymax, src.transform)
```

Comment:
What is `src`? If it's the landscan variable, this code should be in the `with` statement.

Reply:
Ok!